contrib/README.md

```diff
@@ -0,0 +1,344 @@
+# Event Gateway on Kubernetes 
+
+This chart deploys the Event Gateway with etcd onto a Kubernetes cluster. Please note, the default instructions expect
+an existing kubernetes cluster that supports ingress, such as GKE. If your environment doesn't have ingress support
+set up, please follow the [minikube](MINIKUBE.md) instructions to set this up for your development environment.  
+
+## Contents
+
+1. [Quickstart](#quickstart)
+1. [Examples](#examples)
+    1. [Register a function](#register-a-function)
+    1. [Query all function](#query-all-function)
+    1. [Register an event](#register-an-event)
+    1. [Query all events](#query-all-events)
+    1. [Register a subscription](#register-a-subscription)
+    1. [Query all subscriptions](#query-all-subscriptions)
+    1. [Trigger an event](#trigger-an-event)
+1. [Configuration](#configuration)
+1. [Cleanup](#cleanup)
+
+## Quickstart
+
+Make sure you have helm installed on your machine and run `helm init` on your k8s cluster. This will set up the
+`helm` and `tiller` functions required for easy deployment of config files to your cluster. You can follow
+instructions [here](https://docs.helm.sh/using_helm/#quickstart) if you have not set this up previously.
+
+**NOTE:** This portion of the config expects you to have a pre-existing kubernetes cluster (not minikube). For 
+local development please check the [minikube](MINIKUBE.md) information.
```

Comment:
Is this note still needed? As I understand it, the `helm` instructions work with both minikube and something like GKE?